### PR TITLE
added docs for swapfile switch

### PR DESCRIPTION
**Pull request by mikemhenry**
PR Checklist:

- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

This PR is for adding docs if https://github.com/conda-forge/conda-smithy/pull/1751 is merged

### CODE REVIEW

File: src/maintainer/conda_forge_yml.rst [hunk 1]
```diff
@@ -79,6 +79,9 @@ mapping for Azure-specific configuration options. For example:
       # toggle for freeing up some extra space on the default Azure Pipelines
       # linux image before running the Docker container for building
       free_disk_space: False
+      # toggle for creating a 10G swap file on the default Azure Pipelines
+      # linux image before running the Docker container for building
+      make_10G_swap: False
```
Comment (jakirkham) [inline src/maintainer/conda_forge_yml.rst:84]: Wonder if we should allow the size to be customizable. Something like this perhaps?

```suggestion
      make_swap_file: 10Gb
```

**Comment by jakirkham** [comment]
Thanks Mike! 🙏

Had some thoughts on how the configuration could be presented

File: src/maintainer/conda_forge_yml.rst (hunk 1 above)
Comment (mikemhenry) [reply src/maintainer/conda_forge_yml.rst:84]: Yes this is a great idea, the default can be zero and if it is zero it just skips that whole block, but if greater than zero we make a swap file of that size

Comment (jakirkham) [reply src/maintainer/conda_forge_yml.rst:84]: Yeah actually default of `0` makes more sense. Let's go with that

File: src/maintainer/conda_forge_yml.rst [hunk 2]
```diff
@@ -79,9 +79,10 @@ mapping for Azure-specific configuration options. For example:
       # toggle for freeing up some extra space on the default Azure Pipelines
       # linux image before running the Docker container for building
       free_disk_space: False
-      # toggle for creating a 10G swap file on the default Azure Pipelines
+      # Option for creating a swap file on the default Azure Pipelines
       # linux image before running the Docker container for building
-      make_10G_swap: False
+      # for example, use 10G to create a 10G swap file
+      make_swap: 0
```
Comment (jakirkham) [inline src/maintainer/conda_forge_yml.rst:85]:
```suggestion
      make_swap: 0G
```

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Mike! 🙏

**Pull request merged**